themes/default/content/blog/pulumi-security-posture/index.md

```diff
@@ -0,0 +1,97 @@
+
+[Teams](https://www.pulumi.com/docs/intro/pulumi-service/teams/) are groups of Users within an Organization with specific permissions to Stacks.  Consider using these to break up access to environments such as Dev, Staging and Production.  You should look to mirror your code repository or IAM teams/groups as much as possible.
+
+[Organizations](https://www.pulumi.com/docs/intro/pulumi-service/organizations/) are a space for you to collaborate on shared projects and stacks.  You may have multiple Cloud Accounts from a single cloud provider or multiple cloud provider Accounts tied to an Organization in any combination.  It’s recommended that you segment Organizations and Cloud Accounts to limit access and the “blast radius” of security or financial events that may occur within them.
+
+## Security Best Practices with Pulumi
+
+The following recommendations should help you get started and prepare you to scale your company, application and cloud engineering team long term.
+
+## Manage Federated Identity
+
+Connect your user and Pulumi Orgs with an [Identity Provider](https://www.pulumi.com/docs/intro/pulumi-service/organizations/#organization-identity-providers) such as [Github](https://www.pulumi.com/docs/intro/pulumi-service/organizations/#github-identity-provider), [Gitlab](https://www.pulumi.com/docs/intro/pulumi-service/organizations/#gitlab-identity-provider) or [BitBucket](https://www.pulumi.com/docs/intro/pulumi-service/organizations/#bitbucket-identity-provider), your [Single Sign On (SSO)](https://www.pulumi.com/docs/intro/pulumi-service/organizations/#saml-single-sign-on-sso) system and/or another [System for Cross-Domain Identity Management](https://www.pulumi.com/docs/guides/scim/) (SCIM).  Your code repositories hold your Infrastructure as Code crown jewels, and most organizations are motivated to keep access tightly controlled.  SSO improves this process across tools and puts more control into your organization’s hands.  SCIM enables you to manage your users and groups centrally in your Identity Provider (IdP) and then synchronize those users and groups to the Pulumi Service.  Leveraging your existing onboarding and offboarding process will help limit the risk of bad actors in Pulumi itself.
+
+## Build Pulumi Teams
+
+[Teams](https://www.pulumi.com/docs/intro/pulumi-service/teams/#creating-a-team) improve on basic Role Based Access Control for Administrators and Users by allowing you to group users and their access to specific Stacks in your Organization.  For example, a team called ProductionReadOnly could be created and scoped so that all Members have only limited access to read the Production Stacks, while another team, ProductionAdmins, would retain full privileges to those stacks.
```

Comment:
```suggestion
[Teams](https://www.pulumi.com/docs/intro/pulumi-service/teams/#creating-a-team) improve on basic role-based access control (RBAC) for administrators and users by allowing you to group users and their access to specific stacks in your organization.  For example, a team called ProductionReadOnly could be created and scoped so that all members have only limited access to read the production stacks, while another team, ProductionAdmins, would retain full privileges to those stacks.
```